packages/logger/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@lodestar/logger",
+  "description": "Common NodeJS logger for Lodestar binaries",
+  "license": "Apache-2.0",
+  "author": "ChainSafe Systems",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com:ChainSafe/lodestar.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "version": "1.8.0",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js"
+    },
+    "./browser": {
+      "import": "./lib/loggers/browser.js"
+    },
+    "./node": {
+      "import": "./lib/loggers/node.js"
+    },
+    "./empty": {
+      "import": "./lib/loggers/empty.js"
+    }
+  },
```

Comment:
Instead of subpath exports why not not use conditional exports https://nodejs.org/api/packages.html#conditional-exports. And only default export. This way user can use a consistent import pattern. 

```json
"exports": {
    ".": {
      "node": "./lib/loggers/node.js",
      "browser": "./lib/loggers/browser.js",
      "default": "./lib/index.js"
    },
  },
```